rclcpp/test/test_duration.cpp

```diff
@@ -54,6 +54,9 @@ TEST(TestDuration, operators) {
   EXPECT_EQ(sub.nanoseconds(), (rcl_duration_value_t)(young.nanoseconds() - old.nanoseconds()));
   EXPECT_EQ(sub, young - old);
 
+  rclcpp::Duration scale = old * 3;
+  EXPECT_EQ(scale.nanoseconds(), (rcl_duration_value_t)(old.nanoseconds() * 3));
```

Comment:
This should also have tests for the underflow and overflow. In particular I'd like to see at least all 4 cases of underflow and overflow with positive and negative scaling factors.

Reply:
Thanks for comments, they are all reasonable.